### PR TITLE
installation docs: fix table syntax for repos

### PR DESCRIPTION
**Pull request by andresailer**
* fix table syntax for repos
* remove stray quote marks

The dependency table is there now, but the other two tables are gone.

### CODE REVIEW

File: doc/diopside/manual/hardware-installation.rst [hunk 1]
```diff
@@ -161,10 +161,9 @@ You have to configure a dependency repository and either the tag or commit repos
...
 | Alma 8          | ``yum-config-manager --add-repo "https://storage-ci.web.cern.ch/storage-ci/eos/diopside/tag/testing/el-8/x86_64/"``   |
 +-----------------+-----------------------------------------------------------------------------------------------------------------------+
-| Alma 9          | ``yum-config-manager --add-repo "https://storage-ci.web.cern.ch/storage-ci/eos/diopside/tag/testing/el-9/x86_64/"``  |
-|                 |                                                                                                                       |
+| Alma 9          | ``yum-config-manager --add-repo "https://storage-ci.web.cern.ch/storage-ci/eos/diopside/tag/testing/el-9/x86_64/"``   |
 +-----------------+-----------------------------------------------------------------------------------------------------------------------+
-| CentOS9 Stream  | https://storage-ci.web.cern.ch/storage-ci/eos/diopside/tag/testing/fc-38/x86_64/"                                     |
+| CentOS9 Stream  | https://storage-ci.web.cern.ch/storage-ci/eos/diopside/tag/testing/fc-38/x86_64/                                      |
```
Comment (andresailer) [inline doc/diopside/manual/hardware-installation.rst:166]:
```suggestion
| Fedora 38       | https://storage-ci.web.cern.ch/storage-ci/eos/diopside/tag/testing/fc-38/x86_64/                                      |
```

**Comment by esindril** [comment]
Thanks and sorry of that, the display should be fixed now!

**Comment by andresailer** [comment]
Thanks! There is still that CentOS9 Stream vs Fedora 38, but everything is visible now in the docs at least :+1:

**Comment by esindril** [comment]
Eventual consistency ensured! Fixed also the reference to Fedora 38. Thanks!

**Pull request closed without merging**